templates/calico.yaml

```diff
@@ -664,15 +666,34 @@ spec:
             - name: CALICO_IPV4POOL_CIDR
               value: "{{ .Values.podCIDR }}"
             {{- end }}
+            {{- if .Values.podIP4BlockSize }}
+            - name: CALICO_IPV4POOL_BLOCK_SIZE
+              value: "{{ .Values.podIP4BlockSize }}"
+            {{- end }}
+            {{- if .Values.podCIDR6 }}
+            # The default IPv6 pool to create on startup if none exists. Pod IPs will be
+            # chosen from this range. Changing this value after installation will have
+            # no effect. This should fall within `--cluster-cidr`.
+            - name: CALICO_IPV6POOL_CIDR
+              value: "{{ .Values.podCIDR6 }}"
+            {{- end }}
+            {{- if .Values.podIP6BlockSize }}
+            - name: CALICO_IPV6POOL_BLOCK_SIZE
+              value: "{{ .Values.podIP6BlockSize}}"
+            {{- end }}
+            {{- if .Values.podCIDR6 }}
+            - name: CALICO_IPV6POOL_NAT_OUTGOING
+              value: "true"
+            {{- end }}
             # Disable file logging so `kubectl logs` works.
             - name: CALICO_DISABLE_FILE_LOGGING
               value: "true"
             # Set Felix endpoint to host default action to ACCEPT.
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
-            # Disable IPv6 on Kubernetes.
+            # Enable IPv6 on Kubernetes.
```

Comment:
I wonder if IPv6 support should be enabled by default. Is it a non breaking change?